lib/crypto/sig_verifier.go

```diff
@@ -1,26 +1,24 @@
 // Copyright 2021 ChainSafe Systems (ON)
 // SPDX-License-Identifier: LGPL-3.0-only
 
-package runtime
+package crypto
 
 import (
-	"fmt"
 	"sync"
 	"time"
 
 	"github.com/ChainSafe/gossamer/internal/log"
-	"github.com/ChainSafe/gossamer/lib/crypto"
-	"github.com/ChainSafe/gossamer/lib/crypto/ed25519"
-	"github.com/ChainSafe/gossamer/lib/crypto/sr25519"
-	"github.com/ethereum/go-ethereum/crypto/secp256k1"
 )
 
+// SigVerifyFunc verify an signature given a pubkey and msg
+type SigVerifyFunc func(pubkey, sig, msg []byte) (bool, error)
```

Comment:
nit using named returned values for documentation purposes

```suggestion
type SigVerifyFunc func(pubkey, sig, msg []byte) (ok bool, err error)
```